File: sudoku-game/game-boot/src/main/java/com/zumin/sudoku/game/GameBootApplication.java
package com.zumin.sudoku.game;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.cloud.openfeign.EnableFeignClients;

@EnableDiscoveryClient
@EnableFeignClients
@SpringBootApplication
public class GameBootApplication {

  public static void main(String[] args) {
    SpringApplication.run(GameBootApplication.class, args);
  }

}

File: sudoku-game/game-boot/src/main/java/com/zumin/sudoku/game/GameBootApplication.kt
package com.zumin.sudoku.game

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient
import org.springframework.cloud.openfeign.EnableFeignClients

@EnableDiscoveryClient
@EnableFeignClients
@SpringBootApplication
class GameBootApplication

fun main(args: Array<String>) {
  runApplication<GameBootApplication>(*args)
}
